reactive/calico.py

```diff
@@ -48,6 +48,7 @@ def install_calico_binaries():
     log(cmd)
     check_call(cmd)
 
+    # REVIEW(Matt): Is there a suggested source for this tarball?  What versions of the pieces below does it contain?
```

Comment:
Right now there isn't - we manually threw together a tarball while prototyping, and have been using it ever since. So there's definitely a gap there; we'll need to create a build process for this resource.

I don't know which versions we used off the top of my head.